src/components/ui/tech-badge.jsx

```
@@ -0,0 +1,173 @@
+import { motion } from 'framer-motion';
+import { cn } from '@/lib/utils';
+
+const TechBadge = ({ 
+  tech, 
+  className, 
+  size = 'md',
+  glowColor = '#00ffff'
```

ESLint

[error] 'glowColor' is assigned a value but never used.

(no-unused-vars)